collivery.php

```diff
@@ -160,6 +160,59 @@ function mds_collivery_cart_shipping_packages($packages)
         add_filter('woocommerce_cart_shipping_packages', 'mds_collivery_cart_shipping_packages');
     }
 
+
+
+
+    if (!function_exists('mds_collivery_new_order')) {
+        /**
+         * Add the address fields to the order
+         * $order_id is generated by WooCommerce and does not contain the suburb/town names,
+         * use id's to get the names
+         *
+         * @param string $postString
+         *
+         * @return void
+         * @throws WC_Data_Exception
+         */
+        function mds_collivery_new_order($order_id)
```

Comment:
Will this run if the MDS plugin is disabled?
Should this not be wrapped in a `if ($mds->isEnabled()) {`?